scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -167,28 +168,54 @@
                                          (.getName container-status))))
        first))
 
+(defn make-failed-unknown-task-result
```

Comment:
Maybe this should just be `make-failed-task-result`, since it's not always using the `unknown` reason.

Reply:
Renamed.